index.html

```diff
@@ -0,0 +1,59 @@
+<html> 
+
+<head> 
+<title>latex-access, providing easy on-the-fly braille and speech access to LaTeX documents</title> 
+<meta name="keywords" content="latex-access,latex_access,braille,latex,braille translation,math,maths,mathematics,blind,blinndness,scripts,python">
+</head> 
+<body>
+<h1>Latex-access</h1>
+<p>The latex-access project is designed to provide a realtime translation of a line of LaTeX into braille, using either the Nemeth or UEB code, which can be read on a refreshable braille display. This will greatly improve the ease of use of LaTeX to blind mathematicians and scientists.  The project also translates the current line into english speech which is easier to listen to than LaTeX source.</p>
+
+<p>Note that this project is largely aimed at people wishing to read LaTeX using a refreshable braille display and/or speech synthesiser, and people who will probably want to edit LaTeX documents. For example, a university student could receive  worksheets in LaTeX format and produce their work using LaTeX. Using the latex-access scripts, they  will be  able to get a fairly good translation of the question and then an on-the-fly translation of their work as they produce it. If you are not concerned with editing LaTeX documents and simply want a braille translation of an entire laTeX document, then this project is not for you.</p>
+
+<h2>Why the need for the project latex-access?</h2>
+<p>It is widely thought that LaTeX is a good system for a blind mathematician or scientist to use to create and read scientific documents, as it is a linear code and so the user does not have to interact  with two-dimensional notation, such as fractions and column vectors. By reading this linear code, a blind person can take in and understand scientific documents in the same way that a sighted person would do by studying a printed document. It should be noted that normally, laTeX is just a source from which documents are converted into an attractive-looking, typeset document that can be printed or viewed on screen, often in a .pdf, .dvi or .ps format. For various technical reasons, documents in such formats are currently inaccessible with current screen-reading technology. The best current sollution therefore is not to concern ourselves with documents in these formats, but rather to read and interpret the LaTeX source code itself.</p>
```

Comment:
Double l in solution. Obviously this is not introduced by you but it makes sense to clean it up while at it.
```suggestion
<p>It is widely thought that LaTeX is a good system for a blind mathematician or scientist to use to create and read scientific documents, as it is a linear code and so the user does not have to interact  with two-dimensional notation, such as fractions and column vectors. By reading this linear code, a blind person can take in and understand scientific documents in the same way that a sighted person would do by studying a printed document. It should be noted that normally, laTeX is just a source from which documents are converted into an attractive-looking, typeset document that can be printed or viewed on screen, often in a .pdf, .dvi or .ps format. For various technical reasons, documents in such formats are currently inaccessible with current screen-reading technology. The best current solution therefore is not to concern ourselves with documents in these formats, but rather to read and interpret the LaTeX source code itself.</p>
```